### ISSUE TITLE
Unable to generate standalone component with `ionic generate` or `ng generate`

### ISSUE THREAD

**Issue by jerrynavi** [ionic-team/ionic-cli#5073]
**Description:**
Running `ionic generate component <component-name>` or `ng generate component <component-name>` with the `--standalone` flag always throws the error: "Error: Unknown argument: standalone".

**Steps to Reproduce:**
Run `ionic generate component test --standalone` in an Ionic project.

**Output:**
Error: Unknown argument: standalone

**My `ionic info`:**
```
Ionic:

   Ionic CLI                     : 7.1.5 (/opt/homebrew/lib/node_modules/@ionic/cli)
   Ionic Framework               : @ionic/angular 7.6.1
   @angular-devkit/build-angular : 17.0.7
   @angular-devkit/schematics    : 17.0.7
   @angular/cli                  : 17.0.7
   @ionic/angular-toolkit        : 11.0.0

Utility:

   cordova-res                          : 0.15.4
   native-run (update available: 2.0.0) : 1.5.0

System:

   NodeJS : v18.19.0 (/opt/homebrew/Cellar/node@18/18.19.0/bin/node)
   npm    : 10.2.3
   OS     : macOS Unknown
```

**Other Information:**
Nothing to report

**Comment by WMOH-DEV** [comment]
same here, can not generate component 
```
standalone is not defined
```

**Comment by joelcoxokc** [comment]
hey ionic team any updates on this... Generating components is not working

**Comment by dodomui** [comment]
Same here.

**Comment by migueldv90** [comment]
same here

**Comment by migueldv90** [comment]
I was able to fix this issue by updating the angular and ionic libraries.

ng update and npm update

I want to say it was fixed in feat-ng-vite-support #5064

**Comment by dodomui** [comment]
fixed by updated to "@ionic/angular-toolkit": "^11.0.1",

**Comment by jerrynavi** [comment]
Closing as it's been fixed on "@ionic/angular-toolkit": "^11.0.1"

**Comment by forrest-lyman** [comment]
You should reopen this ticket; I just did a fresh build with npx and it installed the angular toolkit 9.x
